### ISSUE TITLE
Scroll Past End

### ISSUE THREAD

**Issue by dmnd** [BrownBear2/atom-overscroll#1]
How is this different from the option `Scroll Past End` in settings?

![image](https://cloud.githubusercontent.com/assets/4427/5641427/a92a3bb4-95ed-11e4-82ab-861400f6b55e.png)

**Comment by BrownBear2** [comment]
Good point. I overlooked that setting, and a quick search didn't turn up anything useful when I looked. I guess this module can be deleted. Thanks for pointing that out.

**Comment by dmnd** [comment]
Oh well, at least you now have the opportunity to compare your own implementation to the one in core!

**Comment by jonleung** [comment]
Is this still in the settings? I can't seem to find it...

**EDIT**: Jk, just found it.

**Comment by jonleung** [comment]
^ Jk just found it ^_^'

**Comment by albirlak** [comment]
For those who have not found:
File -> Settings -> Editor -> [ ] Scroll Past End
![image](https://user-images.githubusercontent.com/31618040/48834278-16e05000-ed86-11e8-991b-a390d4de0bb6.png)
